docusaurus.config.js

```diff
@@ -239,10 +246,10 @@ module.exports = {
           beforeDefaultRemarkPlugins: [versionedLinks],
           // 👋 When cutting a new version, remove the banner for maintained versions by adding an entry. Remove the entry to versions >18 months old.
           versions: {
-            8.1: {
+            8.2: {
               banner: "none",
             },
-            "8.0": {
+            8.1: {
               banner: "none",
             },
           },
```

Comment:
These versions just rotate. 8.0 becomes unsupported, 8.2 is configured to not show the "unsupported" banner.